SpotifyAPI/Local/Models/Track.cs

```diff
@@ -157,5 +157,10 @@ public byte[] GetAlbumArtAsByteArray(AlbumArtSize size)
                 return wc.DownloadData(url);
             }
         }
+
+        public string GetId()
+        {
+            return this.TrackResource.Uri.Substring(14);
```

Comment:
While I think this is a good idea, I would suggest a class called `SpotifyUri` inside `Models` which parses the Uri into it's 3 strings, `Base` (will be always "spotify"), `Type` (like "album" or "track") and `Id`. 

Then we can implement a method inside `TrackResource` called `SpotifyUri ParseUri()` which returns a new instance of `SpotifyUri`.